Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -977,6 +978,40 @@ void UpdatePlanes(std::vector<Frame::Plane::Identifier>& updatedPlanes, std::vec
             }
         }
 
+        void UpdateFeaturePointCloud() {
+            if (!FeaturePointCloudEnabled) {
+                return;
+            }
+
+            ARPointCloud *pointCloud = currentFrame.rawFeaturePoints;
```

Comment:
```suggestion
            ARPointCloud* pointCloud = currentFrame.rawFeaturePoints;
```

Reply:
I think we are about 50/50 on pointer style in this doc, heh.